### DISCUSSION TITLE
Instant NGP crash immediately right after I drop the data folder

### DISCUSSION THREAD

**Discussion by nicale** [NVlabs/instant-ngp#1384] [Q&A]
(Note: I have zero experience in coding and stuff and interested in using this tech for artistic purposes)

![image](https://github.com/NVlabs/instant-ngp/assets/53343165/5db24bbb-d3a6-4485-bf0e-cc04970237f2)
Here is the screenshot of the error right before it crashes, I've tried to find other discussion that have a similar problem, but it still doesn't help since the 1 or 2 discussions that I found I think refer to when one want to build an instant NGP itself instead of using the existing instant NGP based on a certain GPU Card one.

or if I wait bit longer it sometimes stuck and not responding while showing "GLFW 3.4.0 DLL.exe" when I tried to close the "instant-ngp.exe"

I've installed the most recent CUDA and all the things that "requirements.txt" told me to

Here is some information about my PC if that can help:
SYSTEM INFORMATION:
 Operating System: Windows 10 Pro 64-bit (10.0, Build 19045) (19041.vb_release.191206-1406)
  BIOS: BIOS Date: 11/22/16 19:22:13 Ver: 05.0000C (type: BIOS)
                Processor: Intel(R) Core(TM) i3-7100 CPU @ 3.90GHz (4 CPUs), ~3.9GHz
                   Memory: 16384MB RAM
      Available OS Memory: 16336MB RAM
      DirectX Version: DirectX 12
      
 DISPLAY DEVICES:
  Card name: NVIDIA GeForce GTX 1050 Ti
        Manufacturer: NVIDIA
           Chip type: NVIDIA GeForce GTX 1050 Ti
            DAC type: Integrated RAMDAC
         Device Type: Full Device (POST)
          Device Key: Enum\PCI\VEN_10DE&DEV_1C82&SUBSYS_00007377&REV_A1
       Device Status: 0180200A [DN_DRIVER_LOADED|DN_STARTED|DN_DISABLEABLE|DN_NT_ENUMERATOR|DN_NT_DRIVER] 
 Device Problem Code: No Problem
 Driver Problem Code: Unknown
      Display Memory: 12185 MB
    Dedicated Memory: 4017 MB
       Shared Memory: 8168 MB
        Current Mode: 1440 x 900 (32 bit) (60Hz)
         HDR Support: Not Supported
    Display Topology: Internal
 Display Color Space: DXGI_COLOR_SPACE_RGB_FULL_G22_NONE_P709
     Color Primaries: Red(0.630859,0.348633), Green(0.340820,0.622070), Blue(0.152344,0.057617), White Point(0.313477,0.329102)
   Display Luminance: Min Luminance = 0.500000, Max Luminance = 270.000000, MaxFullFrameLuminance = 270.000000
   
   
Let me know if I need to provide other information in order to solve this issue, thanks in advance!

**Comment by QXmX29** [comment]
I've met a similar problem... Have you solved it?

**Comment by pranavsys** [comment]
I got the same problem. Please know if you get a solution.

**Reply by pranavsys** [reply]
Let me know*

**Comment by KjMonge** [comment]
I am having a similar problem. It seems related to the data set as it only crashes with certain sets.

**Comment by KjMonge** [comment]
Turns out it was a problem with the .json file. The file paths were incorrect. I replaced them with the correct locations and the crashes stopped.

**Reply by newarsunder** [reply]
which file paths? can you please elaborate. I checked and there were some missing images in transforms.json and in images folder, so i matched them accordingly but still it crashes.

**Comment by haoyenlu** [comment]
I had the same problem and I found out that the **file path of each frame in the transforms.json** is not correct, probably because I put the image in another folder when using colmap. I fixed this problem by simply changing the file path to the correct one, for instance, I changed the file path from "videos/images/0001.jpg" to "images/0001.jpg", and the problem is fixed,
